docs/usage/custom-dns-config.md

```diff
@@ -38,6 +38,9 @@ data:
          whoami
 ```
 
+The port number 8053 in `global:8053` is the specific port that CoreDN is bound to and cannot be changed to any other port. 
```

Comment:
```suggestion
The port number 8053 in `global:8053` is the specific port that CoreDNS is bound to and cannot be changed to any other port if it should act on ordinary name resolution requests from pods. Otherwise, CoreDNS will open a second port, but you are responsible to direct the traffic to this port. `kube-dns` service in `kube-system` namespace will direct name resolution requests within the cluster to port 8053 on the CoreDNS pods.
```